felles/src/main/kotlin/no/nav/familie/kontrakter/felles/oppgave/FinnOppgaveRequest.kt

```diff
@@ -1,18 +1,29 @@
 package no.nav.familie.kontrakter.felles.oppgave
 
-class FinnOppgaveRequest(val tema: String? = null,
-                         val behandlingstema: String? = null,
-                         val oppgavetype: String? = null,
-                         val enhet: String? = null,
-                         val saksbehandler: String? = null,
-                         val journalpostId: String? = null,
-                         val tilordnetRessurs: String? = null,
-                         val tildeltRessurs: Boolean? = null,
-                         val opprettetFomTidspunkt: String? = null,
-                         val opprettetTomTidspunkt: String? = null,
-                         val fristFomDato: String? = null,
-                         val fristTomDato: String? = null,
-                         val aktivFomDato: String? = null,
-                         val aktivTomDato: String? = null,
-                         val limit: Long? = null,
-                         val offset: Long? = null)
+import no.nav.familie.kontrakter.felles.abstraction.QueryObject
+import org.springframework.format.annotation.DateTimeFormat
```

Comment:
Hvorfor import org.springframework.format.annotation.DateTimeFormat ?

Reply:
Siden dette skal konverteres fra en query-string, så er det ikke json og det er springrammeverket som må gjøre jobben. Spring må ha litt mer hjelp for å lese en String inn i LocalDate.